stdlib/public/core/Random.swift

```diff
@@ -17,7 +17,7 @@ import SwiftShims
 /// When you call methods that use random data, such as creating new random
 /// values or shuffling a collection, you can pass a `RandomNumberGenerator`
 /// type to be used as the source for randomness. When you don't pass a
-/// generator, the default `Random` type is used.
+/// generator, the default `SystemRandomNumberGenerator` type is used.
```

Comment:
There's another of these (`` `Random` `` => `` `SystemRandomNumberGenerator` ``) on line 44, under the `Conforming to the RandomNumberGenerator Protocol` heading.